src/Pokemon.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace Pokedex;
+
+class Pokemon
+{
+
+    private $id;
+    private $name;
+    private $type_1;
+    private $type_2;
+
+
+    public function __construct()
+    {
+        //capitalises first letter of each string for more a professional appearance
+        $this->name = ucfirst($this->name);
+        $this->type_1 = ucfirst($this->type_1);
+        $this->type_2 = ucfirst($this->type_2);
+    }
+
+    /**
+     * @return int The pokemon's ID.
+     */
+    public function getId()
```

Comment:
Getters could all be return type hinted.

Reply:
Solved in eb4d541

Reply:
Solved in eb4d541